server/ai_live_video.go

```diff
@@ -496,6 +497,20 @@ func copySegment(ctx context.Context, segment *http.Response, w io.Writer, seq i
 	}
 }
 
+func setOutWriter(ctx context.Context, writer *media.RingBuffer, params aiRequestParams) {
+	params.node.LiveMu.Lock()
+	defer params.node.LiveMu.Unlock()
+	stream, requestID := params.liveParams.stream, params.liveParams.requestID
+	sess, exists := params.node.LivePipelines[stream]
+	if !exists || sess.RequestID != requestID {
+		clog.Info(ctx, "Did not set output writer due to nonexistent stream or mismatched request ID")
```

Comment:
Since we have the check and log, maybe specify in the log which one happened?

```suggestion
		clog.Info(ctx, "Did not set output writer due to nonexistent stream or mismatched request ID", "exists", exists)
```

Reply:
Improved the logging in 556bd849ed9fbceb9463ade375ca395360024489